compiler-rt/test/rtsan/test_rtsan.cpp

```diff
@@ -0,0 +1,17 @@
+// RUN: %clangxx -fsanitize=realtime %s -o %t
```

Comment:
The directory is about test/rtsan, so any word like "test" "rtsan" does not carry more information.

You can call it `basic.cpp` if you think the readers can start reading this test first.